### PR TITLE
Admin collector

### PR DESCRIPTION
**Pull request by mafrosis**
A new collector!

Simply wraps this API:
https://docs.github.com/en/enterprise-server@2.22/rest/reference/enterprise-admin#admin-stats

Only contentious issue might be formatting - it was rather verbose and frankly just an enormous pile of boilerplate wrapping the go-github lib. So I made it more concise by removing a lot of line breaks..

### CODE REVIEW

**Comment by tboerger** [comment]
I'm waiting for a developer license for GutHub Enterprise, I would like to give it a try first.

File: pkg/exporter/adminstats.go [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+	// comments
+	CommentsCommitTotal      *prometheus.Desc
+	CommentsGistTotal        *prometheus.Desc
+	CommentsIssueTotal       *prometheus.Desc
+	CommentsPullRequestTotal *prometheus.Desc
+}
+
+// NewAdminStatsCollector returns a new AdminStatsCollector.
+func NewAdminStatsCollector(logger log.Logger, client *github.Client, failures *prometheus.CounterVec, duration *prometheus.HistogramVec, cfg config.Target) *AdminStatsCollector {
+	failures.WithLabelValues("org").Add(0)
```
Comment (mafrosis) [inline pkg/exporter/adminstats.go:70]: I noticed this should be `adminstats`. Let me know if you have any other feedback before I push a fix

**Comment by tboerger** [comment]
Sadly GitHub never came back to me with a developer license :(

**Comment by tboerger** [comment]
@mafrosis thanks for your effort, finally I have integrated the admin stats, sadly you have removed the repo fork and I haven't been able to cherry-pick your commits. You can see the result at https://github.com/promhippie/github_exporter/pull/184, currently it's only part of `latest` image, but it will be part of a 2.0.0 release.

**Pull request closed without merging**